internal/armada/server/submit_from_log.go

```diff
@@ -113,8 +113,7 @@ func (srv *SubmitFromLog) Run(ctx context.Context) error {
 			// If this message isn't for us we can simply ack it
 			// and go to the next message
 			if !schedulers.ForLegacyScheduler(msg) {
-				srv.Consumer.Ack(msg)
-				break
+				srv.ack(ctx, msg)
```

Comment:
Is removing the break here intentional?

Reply:
no- it wasn't.  @theAntiYeti can we put this back

Reply:
done